source/clear-linux/guides/maintenance/swupd-mixer-integration.rst

```diff
@@ -0,0 +1,117 @@
+.. _swupd_mixer_integration:
```

Comment:
The label must be identical to the file name, do not use underscores.
swupd-mixer-integration